jwst/ramp_fitting/ramp_fit.py

```diff
@@ -1193,6 +1550,12 @@ def gls_ramp_fit(input_model, buffsize, save_opt, readnoise_model, gain_model, m
     gain_model : instance of gain model
         Gain for all pixels.
 
+    max_cores : string
+        Number of cores to use for multiprocessing. If set to 'none' (the default),
+        then no multiprocessing will be done. The other allowable values are 'quarter',
+        'half', and 'all'. This is the fraction of cores to use for multi-proc. The
+        total number of cores includes the SMT cores (Hyper Threading for Intel).
+
```

Comment:
Does `gls_ramp_fit` actually allow multiprocessing?  I didn't think it did.

Reply:
I didn't add the parameter to the function.  I simply added to the doctoring to capture all the parameters.  I have not looked at this function.  The only reason I noticed it is because I created the function `compute_slices`.